tests/test_epsagon_init.py

```diff
@@ -144,6 +144,7 @@ def test_epsagon_wrapper_env_init(wrapped_get):
     'EPSAGON_LOGGING_TRACING_ENABLED': 'TRUE',
     'AWS_LAMBDA_FUNCTION_NAME': None,
     'EPSAGON_STEPS_OUTPUT_PATH': '',
+    'EPSAGON_SAMPLE_RATE': 0.5
```

Comment:
Extract 0.5 here and in the test assertion to a const (making it easier to understand the connection between the two)

Reply:
These tests are pretty straightforward... values in the mock object should match the values called in the init function...
All other parameters are not stored as consts as well (and I also think it should stay like that since these are tests)